addon/tracker.js

```diff
@@ -0,0 +1,72 @@
+/* global Ember */
+
+
+export default {
+  build: function(addon) {
+
+    var tracker           = this.tracker(addon.settings.gaGlobalFuncName);
```

Comment:
It seems that `addon.settings` is `null` at the moment this code is executed.
So I get an error in console: `Uncaught TypeError: Cannot read property 'gaGlobalFuncName' of null`